### PR TITLE
CNCF Announcement

### PR DESCRIPTION
**Pull request by carolynvs**
# What does this change
Top secret blog post announcing that we are in the CNCF

https://deploy-preview-1303--porter.netlify.app/blog/cncf/

# What issue does it fix
Tell people who obviously don't follow our GitHub pull requests 😉 

# Notes for the reviewer
Here was some of the guidance I was given for the announcement:

example of previous announcement: https://openservicemesh.io/blog/open-service-mesh-osm-accepted-into-cncf-as-a-sandbox-project/

General Structure of Announcements:
* the announcement
* a little bit of history
* where you hope to go from here
* maybe some acknowledgements
* call to actions if any

# Checklist
- [ ] Unit Tests
- [ ] Documentation
- [ ] Schema (porter.yaml)

### CODE REVIEW

File: docs/content/blog/cncf.md
Comment (iennae) [inline docs/content/blog/cncf.md:33]: If you are interested in learning more about Porter, check out this introductory video. Learn more about the [Porter Community](https://porter.sh/community/) and contribute to the [project](https://github.com/getporter/porter/blob/main/CONTRIBUTING.md).

File: docs/content/blog/cncf.md
Comment (iennae) [inline docs/content/blog/cncf.md:25]: This feels like we need a whole separate post for the intro to the Michelle the Porter Cat.

File: docs/content/blog/cncf.md
Comment (iennae) [inline docs/content/blog/cncf.md:18]: When Jeremy Rickard and I created Porter less than two years ago, we wanted to create a community-led project that would encourage collaboration across vendors to make Cloud Native Application Bundles (CNAB) accessible and maintainable. In joining the CNCF, we can continue to support that mission of vendor neutral independence while growing the community of practitioners who can use and contribute to Porter's development.

Comment (iennae) [reply docs/content/blog/cncf.md:18]: Today, the Porter Maintainers are happy to announce that Porter is now an official CNCF Sandbox project. By contributing Porter to the CNCF, we hope to encourage the adoption of Porter to ease the creation and use of bundles and to see more companies contribute mixins.

File: docs/content/blog/cncf.md
Comment (iennae) [inline docs/content/blog/cncf.md:20]: projects to have a public venue where we can continue to grow and experiment with a goal of having the best developer experience working with bundles. We welcome contributions from everyone in the community and provide a contribution ladder to encourage and support contributors.

File: docs/content/blog/cncf.md
Comment (iennae) [reply docs/content/blog/cncf.md:33]: In the near-term, we are focusing on [FILL IN BLANK FROM ROADMAP]. 

In addition to contributing ideas and energy to those initiatives, developers can contribute mixins using [skeletor](https://github.com/getporter/skeletor) as a starting point. Examples of mixins include the [Docker Mixin](https://github.com/getporter/docker-mixin) and [Terraform Mixin](https://github.com/getporter/terraform-mixin).

File: docs/content/blog/cncf.md
Comment (karenhchu) [inline docs/content/blog/cncf.md:11]: Do you want to add a link to his Twitter or Github account?

**Comment by carolynvs** [comment]
@karenhchu I got an email with one of your comments but now it's gone? The latest draft has a link to Jeremy's github profile now.

**Review by iennae — APPROVED** [review]
LGTM :D

File: docs/content/blog/cncf.md
Comment (karenhchu) [inline docs/content/blog/cncf.md]:
```suggestion
Porter becoming a CNCF [sandbox] project is the first step for early stage
```

File: docs/content/blog/cncf.md
Comment (karenhchu) [inline docs/content/blog/cncf.md:9]:
```suggestion
across vendors to make [Cloud Native Application Bundles (CNAB)] [CNAB] accessible and
```

Comment (karenhchu) [inline docs/content/blog/cncf.md:9]: probably worth hyperlinking to CNAB - https://cnab.io/

**Review by karenhchu — COMMENTED** [review]
Updating comments -- my last set of comments didn't get committed before your updates.

File: docs/content/blog/cncf.md
Comment (carolynvs) [reply docs/content/blog/cncf.md:4]: Thanks for the reminder!

**Review by vdice — APPROVED** [review]
🎉

**Comment by carolynvs** [comment]
Holding off on merging until https://github.com/getporter/porter/pull/1304 is merged so that it looks pretty.

**Comment by jeremyrickard** [comment]
LGTM

**Review by jeremyrickard — APPROVED** [review]

**Comment by carolynvs** [comment]
Rebased with the author changes.

**Pull request merged**
